simple-page-ordering.php

```diff
@@ -125,25 +139,17 @@ public static function ajax_simple_page_ordering() {
 				die( - 1 );
 			}
 
-			// do we have a nonce that verifies?
-			if ( empty( $_POST['_wpnonce'] ) || empty( $_POST['screen_id'] ) ) {
-				// no nonce to verify...
-				die( -1 );
-			} else {
-				check_admin_referer( 'simple-page-ordering_' . sanitize_text_field( wp_unslash( $_POST['screen_id'] ) ) );
-			}
-
-			// badly written plug-in hooks for save post can break things
+			// Badly written plug-in hooks for save post can break things.
 			if ( ! defined( 'WP_DEBUG' ) || ! WP_DEBUG ) {
-				error_reporting( 0 );
+				error_reporting( 0 ); // phpcs:ignore WordPress.PHP.DiscouragedPHPFunctions.runtime_configuration_error_reporting
 			}
 
 			global $wp_version;
 
 			$previd   = empty( $_POST['previd'] ) ? false : (int) $_POST['previd'];
 			$nextid   = empty( $_POST['nextid'] ) ? false : (int) $_POST['nextid'];
 			$start    = empty( $_POST['start'] ) ? 1 : (int) $_POST['start'];
-			$excluded = empty( $_POST['excluded'] ) ? array( $post->ID ) : array_filter( (array) json_decode( $_POST['excluded'] ), 'intval' );
+			$excluded = empty( $_POST['excluded'] ) ? array( $post->ID ) : array_filter( (array) json_decode( $_POST['excluded'] ), 'intval' ); // phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
```

Comment:
There is no way to sanitize the incoming parameter before the `json_decode` function. The input is immediately sanitized after converting it to an array.

Reply:
I don't think that it exists @dinhtungdu 

https://developer.wordpress.org/reference/functions/wp_json_decode